pkg/sysregistries/system_registries.go

```diff
@@ -97,3 +97,16 @@ func GetInsecureRegistries(sys *types.SystemContext) ([]string, error) {
 	}
 	return config.Registries.Insecure.Registries, nil
 }
+
+// RegistriesConfPath is the path to the system-wide registry configuration file
+func RegistriesConfPath(ctx *types.SystemContext) string {
```

Comment:
Please make `readRegistryConf` call this instead of having two copies of the code that will almost certainly get out of sync.

Reply:
Fixed